### PR TITLE
Adds Synthetic Manufacturers 

### PR DESCRIPTION
**Pull request by Steelpoint**
# About the pull request

Allows Synthetic players to select a manufacturer for their Synthetic character.

This currently does not have any tangible in-game effect outside of allowing a Synthetic player to inform themselves of who manufactured them. 

In addition to a broad suite of manufactures, all of which have some basis in Aliens lore (noted in the photos below), I added a 'independent manufacturer' for edge cases. Currently players will default to Weyland-Yutani if they don't select anything. 

# Explain why it's good for the game

Adds a little bit of lore and flavour to a Synthetic players character building.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Screenshot 2023-10-15 13 53 32

</details>

# Changelog
:cl:
add: Synthetics may now select their manufacturer. This currently has no tangible in-game effect. 
/:cl:

### CODE REVIEW

**Comment by Steelpoint** [comment]
Requested changes made.

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -1611,6 +1614,11 @@ var/const/MAX_SAVE_SLOTS = 10
 					if(new_relation)
 						nanotrasen_relation = new_relation
 
+				if("synth_manufacturer")
+					var/synth_builder = input(user, "Choose your manufacturer.")  as null|anything in list("Weyland-Yutani", "Borgia Industries", "Hyperdyne Systems", "Lasalle Bionational", "Seegson", "MedTech", "Grant Corporation", "AlphaTech Hardware", "Independent Manufacturer")
```
Comment (harryob) [inline code/modules/client/preferences.dm:1618]: tgui_input_list

File: code/modules/client/preferences_savefile.dm [hunk 2]
```diff
@@ -242,6 +243,7 @@
 	adaptive_zoom = sanitize_integer(adaptive_zoom, 0, 2, 0)
 	tooltips = sanitize_integer(tooltips, FALSE, TRUE, TRUE)
 
+	if(isnull(synth_manufacturer)) synth_manufacturer = initial(synth_manufacturer)
```
Comment (harryob) [inline code/modules/client/preferences_savefile.dm:246]: this should use sanitize_inlist() with the above global list

File: code/modules/client/preferences.dm (hunk 1 above)
Comment (harryob) [inline code/modules/client/preferences.dm:1618]: this list should be a global list or define so it can be referenced later for sanitizing

**Review by harryob — CHANGES_REQUESTED** [review]

**Pull request closed without merging**